plan/physical.go

```diff
@@ -112,12 +121,50 @@ func OnlyPhysicalRules(rules ...Rule) PhysicalOption {
 	})
 }
 
-func DisableValidatation() PhysicalOption {
+func DisableValidation() PhysicalOption {
 	return physicalOption(func(p *physicalPlanner) {
 		p.disableValidation = true
 	})
 }
 
+// physicalConverterRule rewrites logical nodes that have a ProcedureSpec that implements
+// PhysicalProcedureSpec as a physical node.  For operations that have a 1:1 relationship
+// between their physical and logical operations, this is the default behavior.
+type physicalConverterRule struct {
+}
+
+func (physicalConverterRule) Name() string {
+	return "physicalConverterRule"
+}
+
+func (physicalConverterRule) Pattern() Pattern {
+	return Any()
+}
+
+func (physicalConverterRule) Rewrite(pn PlanNode) (PlanNode, bool, error) {
+	if _, ok := pn.(*PhysicalPlanNode); ok {
+		// Already converted
+		return pn, false, nil
+	}
+
+	ln := pn.(*LogicalPlanNode)
+	pspec, ok := ln.Spec.(PhysicalProcedureSpec)
+	if !ok {
+		// A different rule will do the conversion
```

Comment:
So currently if we add a logical procedure that is not a physical procedure, the rewrite won't happen and and error will occur in the physical planner?

Reply:
Yes, this is exactly why I had to add the default cost methods to generated yields.  Lots of the end-to-end tests that didn't have explicit yields were failing to plan.